[doc][core][cgraph] Use with_tensor_transport API (#50273)
ray-project/ray: doc/source/ray-core/compiled-graph/quickstart.rst

.. testcode::

    pip install "ray[cg]"
    

>     # Note that before ray version 2.41, use the following instead:

Reviewer: ```suggestion
    # For any ray version before 2.41, use the following instead:
```
